docs/en/integrations/data-ingestion/kafka/msk/index.md

```diff
@@ -61,3 +61,15 @@ consumer.max.partition.fetch.bytes=1048576
 
 You can find more details (both implementation and other considerations) in the official [Kafka](https://kafka.apache.org/documentation/#consumerconfigs) and 
 [Amazon MSK](https://docs.aws.amazon.com/msk/latest/developerguide/msk-connect-workers.html#msk-connect-create-custom-worker-config) documentation.
+
+## Private Link Setup
+When integrating ClickHouse with Amazon MSK using AWS PrivateLink, it's important to ensure that your private VPC (Virtual Private Cloud) has access to the internet to reach your ClickHouse instance.
```

Comment:
@tpanetti can you clarify what kind of setup we describe here?

MSK with a connector will be establishing the connection to ClickHouse Cloud. If we want to establish a private link between customers' VPC and ClickHouse Cloud, this approach should be follow: https://clickhouse.com/docs/en/manage/security/aws-privatelink

Why do we mention and require configuring internet access from customers' VPC? MSK connector will connect via private DNS endpoint.

Reply:
I added this section as this is currently our workaround to allow Private Link to use MSK. Entirely routing through Private Link results in a DNS resolution error that is currently not fixed (More details [here](https://clickhouse-inc.slack.com/archives/C041CL09ABE/p1699404515003859?thread_ts=1698948609.985519&cid=C041CL09ABE) . Your comment makes sense though,  I'm changing this section to general networking, should I add a blurb about private link or wait until we have resolution?